website/urls.py

```diff
@@ -22,17 +22,12 @@
 from django.contrib import admin
 from django.urls import include, path
 
-
 localized_urls = i18n_patterns(
     path("", include("codeguru.urls")),
     path("challenges/", include("war.urls")),
 )
 
-urlpatterns = [
-    *localized_urls,
-    path("admin/", admin.site.urls),
-    path("-/", include("django_alive.urls"))
-]
+urlpatterns = [*localized_urls, path("admin/", admin.site.urls), path("-/", include("django_alive.urls"))]
```

Comment:
nitpick: add a comma at the end of the list so black puts each argument in a separate line